src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../api";
+import { useParams } from "react-router";
+import SingleArticleCard from "./SingleArticleCard";
+
+export default function SingleArticle() {
+  const [currentArticle, setCurrentArticle] = useState([]);
```

Comment:
if a currentArticle is an object thats what we should define as the starting value just empty.